edit_articles.js

```diff
@@ -0,0 +1,95 @@
+
+  webDB.reconnect();
+
+  var editor = {};
+  var chosenID = 0;
+
+  //Gets article ID from url
+  editor.getArticleID = function (){
+    chosenID = window.location.search.substring(4);
+    return chosenID;
+  };
+
+  //Displays article retrieved from database
+  editor.displayPreview = function (resultObject){
+    var entry = {};
+    entry.title = resultObject[0].title;
+    entry.author = resultObject[0].author;
+    entry.authorUrl = resultObject[0].authorUrl;
+    entry.category = resultObject[0].category;
+    entry.body = marked(resultObject[0].markdown);
+    entry.publishedOn = resultObject[0].publishedOn;
+
+    $.get('templates/article.handlebars', function(result) {
+      var previewTemplateScript = $('#article-template').html();
+      var previewTemplate = Handlebars.compile(previewTemplateScript);
+      var previewHTML = previewTemplate(entry);
+
+      $('#preview').empty().append(previewHTML);
+
+      $('pre code').each(function (i, block){
+        hljs.highlightBlock(block);
+      });
+      var newArticle = JSON.stringify(entry);
+      $('#jsonString').val(newArticle);
+    });
+  };
+
+  //Gets article from database based on article ID in url
+  editor.displayChosenArticle = function () {
+    webDB.execute(
+      'SELECT title, author, authorUrl, category, publishedOn, markdown FROM articles WHERE id=' + editor.getArticleID() + ';',
+      function(results){
+        $('#articleTitle').val(results[0].title);
+        $('#author').val(results[0].author);
+        $('#authorUrl').val(results[0].authorUrl);
+        $('#category').val(results[0].category);
+        $('#articleBody').html(results[0].markdown);
+        editor.displayPreview(results);
+      });
+  };
```

Comment:
What happens when there's no ID in the url? Your console log should be showing you the answer when this gets executed in index.html.